rails-commenting.rb

```diff
@@ -6,25 +6,25 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# 1)
+# 1) Defining the the current ruby controller class and inheriting methods from the parent controller.
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2)An instance object that is equal to every single blogpost. 
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3)An instance variable that is equal to the blog post you want with the same Id number you pass.
     @post = BlogPost.find(params[:id])
   end
 
-  # 4)
+  # 4)Creating a method called new that can create a new model instance.
```

Comment:
Yes, in a round about way. The new method is really setting up the form. It is easy to confuse this method with the create methods, but they actually perform two different, yet interconnected tasks.